tests-e2e/security_test.go

```diff
@@ -117,15 +118,9 @@ func TestSecurityLiveStreamEventLeftLeak(t *testing.T) {
 			Sender: alice.UserID,
 			ID:     sensitiveEventID,
 		},
-	}, lastTwoEvents)
-	sensitiveEvent := lastTwoEvents[1]
-	kickEvent := lastTwoEvents[0]
-
-	// TODO: WE should be returning updated values for name and required_state
-	m.MatchResponse(t, aliceRes, m.MatchList("a", m.MatchV3Count(1)), m.MatchNoV3Ops(), m.MatchRoomSubscription(
-		roomID, m.MatchRoomTimelineMostRecent(2, []json.RawMessage{kickEvent, sensitiveEvent}),
-	))
```

Comment:
The point being here: we don't care if the events arrive in one sync response or two?

Reply:
Yes